src/containers/Bookings.js

```diff
@@ -1,19 +1,79 @@
 import React, { Component } from "react";
-import Search from "../components/Search.js";
-// import Results from "../components/Results.js";
-// import * as FakeBookings from "../data/fakeBookings.json";
+import Search from "../components/Search/Search.js";
+import Results from "../components/Results/Results.js";
+import FakeBookings from '../data/fakeBookings.json'
+
 
 export default class Bookings extends Component {
+  state = {
```

Comment:
This looks like next week's homework! I'm going to leave off reviewing it for now, maybe you can check it once we've gone over props & state in the class.